app/qml/map/Highlight.qml

```diff
@@ -216,19 +205,9 @@ Item {
     constructHighlights()
   }
 
-  onGuideLineAllowedChanged: {
-    if ( guideLineAllowed )
-      constructHighlights()
-    else updateGuideLine()
-  }
-
-  onPositionChanged: {
-    if ( !recordingInProgress )
```

Comment:
it looks like here are 2 changes: 1) simple fix of CRS 2) some cleaning (construct + update ===> constuct only). 

1) looks OK
2) I am not able to asses if the previous and new flow is the same ... (looks like now we do always some extra checks on l 122)